src/component/video-card/VideoCard.jsx

```diff
@@ -0,0 +1,21 @@
+import './video-card.css';
+import MoreVertOutlinedIcon from '@mui/icons-material/MoreVertOutlined';
+
+const VideoCard = ({video}) => {
```

Comment:
suggestion: you can destructure this video prop further so you don't have to type video.propertyName all the time.